tools/clang/test/CodeGenSPIRV/intrinsics.get-attribute-at-vertex.cbuf.var.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: dxc -T ps_6_2 -E main -spirv %s | FileCheck %s
```

Comment:
Should be `RUN: %dxc`. Same applies for a few more tests below.

Reply:
Ah yes, fixed!